### PR TITLE
Symbol Color Changable

### PR DESCRIPTION
**Pull request by Rhotav**
I couldn't find this feature while working on a project, then I had to make changes to Internal. Now we can change the symbol colors with the "DoneSymbolColor" and "PromptSymbolColor" properties on the ColorSchema class.
Screenshot_1

### CODE REVIEW

File: Sharprompt/Internal/OffscreenBuffer.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public OffscreenBuffer(IConsoleDriver consoleDriver)
         }
 
         private readonly IConsoleDriver _consoleDriver;
-        private readonly List<List<TextInfo>> _outputBuffer = new();
+        private readonly List<List<TextInfo>> _outputBuffer = new List<List<TextInfo>>();
```
Comment (shibayan) [inline Sharprompt/Internal/OffscreenBuffer.cs:19]: This change is not necessary.

File: Sharprompt/Prompt.Customize.cs [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+using System;
 
 namespace Sharprompt
 {
     public static partial class Prompt
     {
         public static class ColorSchema
         {
+            public static ConsoleColor DoneSymbolColor { get; set; } = ConsoleColor.Green;
+            public static ConsoleColor PromptSymbolColor { get; set; } = ConsoleColor.Green;
```
Comment (shibayan) [inline Sharprompt/Prompt.Customize.cs:10]: Remove the `Color` suffix.

```suggestion
            public static ConsoleColor DoneSymbol { get; set; } = ConsoleColor.Green;
            public static ConsoleColor PromptSymbol { get; set; } = ConsoleColor.Green;
```

**Review by shibayan — CHANGES_REQUESTED** [review]

**Comment by shibayan** [comment]
Thanks @Rhotav. Naming conventions and C# code formatting will need to be changed.
The code format can be modified by running the `dotnet format` command.

File: Sharprompt/Internal/OffscreenBuffer.cs (hunk 1 above)
Comment (Rhotav) [reply Sharprompt/Internal/OffscreenBuffer.cs:19]: Upps I think this change was made automatically by Visual Studio.

**Review by shibayan — CHANGES_REQUESTED** [review]
The status check error needs to be resolved.

**Comment by Rhotav** [comment]
@shibayan I don't quite understand what to do? (I'm making a pull request for the first time, sorry)

**Comment by shibayan** [comment]
@Rhotav This can be fixed by running `dotnet format` in the Sharprompt directory after installing the [dotnet-format](https://github.com/dotnet/format#how-to-install) tool.

```sh
dotnet tool install -g dotnet-format
dotnet format
```

Here's an example of how it works (in Japanese)
image

**Comment by Rhotav** [comment]
@shibayan Okay, I got it and did it. thank you for your help

**Review by shibayan — APPROVED** [review]
LGTM 👍

**Pull request merged**